presto-main/src/main/java/com/facebook/presto/metadata/Metadata.java

```diff
@@ -130,6 +130,11 @@
     void renameTable(Session session, TableHandle tableHandle, QualifiedObjectName newTableName);
 
     /**
+     * Comments to the specified table.
+     */
+    void commentTable(Session session, TableHandle tableHandle, String comment);
```

Comment:
Can we name this sth like `addComment` (similar to `addColumn`)? Or maybe `commentOn`?

BTW `addComment` sounds like we can add multiple comments to a table, will any connector support that?